### PR TITLE
Release 1.6 vivek

### PR DESCRIPTION
**Pull request by vivekbhr**
Fixed bam2nucleosome installation and execution problem.
Edited the docs with examples and images.

### CODE REVIEW

File: docs/content/tools/plotCorrelation.rst [hunk 1]
```diff
@@ -1,8 +1,33 @@
 plotCorrelation
-===============
+==========================
 
 .. argparse::
-   :ref: deeptools.plotCorrelation.parse_arguments
-   :prog: plotCorrelation
+   :ref: deeptools.bamFingerprint.parse_arguments
+   :prog: bamFingerprint
```
Comment (bgruening) [inline docs/content/tools/plotCorrelation.rst]: @vivekbhr is this correct? We are in plotCorrelation here.

Comment (vivekbhr) [inline docs/content/tools/plotCorrelation.rst]: @bgruening sorry forgot to edit the argparse line.. It's plotCorrelation.. Fixed now.

File: bin/bam2nucleosome [hunk 2]
```diff
@@ -39,7 +40,11 @@ def parseArguments(args=None):
                         default = None,
                         type = int,
                         required = False)
-
+    parser.add_argument('--smoothLength',
+                        help = "length of smoothing to be done.",
```
Comment (bgruening) [inline bin/bam2nucleosome]: Capital "L"

**Comment by bgruening** [comment]
:+1:

File: bin/bam2nucleosome (hunk 2 above)
Comment (vivekbhr) [inline bin/bam2nucleosome]: I think help line will have to be elaborated a bit on this arg, but I left it to fidel..

**Pull request merged**